.github/workflows/stale.yml

```
@@ -0,0 +1,19 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
```

I prefer to use the latest version of it.
```suggestion
      - uses: actions/stale@main
```